sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -118,6 +118,13 @@ private[spark] object HiveUtils extends Logging {
     .toSequence
     .createWithDefault(Nil)
 
+  val HIVE_METASTORE_METACONF_PREFIX = buildStaticConf("spark.sql.hive.metastore.metaconf.prefix")
+    .doc("the prefix of the meta conf in HiveMetastoreClient, which is used to extract " +
```

Comment:
This sounds misleading because this PR calls `c.setMetaConf` instead of extracting. Can we use more direct terms to describe what this conf aims?

Reply:
I want to update the description to "prefix of the meta conf that could be used to set the HiveMetastoreClient.", do you think it's better?

Reply:
> This sounds misleading because this PR calls `c.setMetaConf` instead of extracting. Can we use more direct terms to describe what this conf aims?

cause we couldn't get the meta conf directly  and  the prefix is used to extract meta conf from HiveConf.

Reply:
I will add detailed description for the configuration

Reply:
Can you show a self-contained working example in the Pr description?

Reply:
ok, i add a example about how to use the additional conf in this PR.  pls check